js/util/evented.js

```diff
@@ -78,7 +78,14 @@ var Evented = {
      * @returns {Object} `this`
      */
     fire: function(type, data) {
-        if (!this.listens(type)) return this;
+        if (!this.listens(type)) {
+            // Error events are important for debugging and should not be
+            // silently dropped.
+            if (util.endsWith(type, 'error')) {
+                (console.trace || console.error)('Dropped "error" event: ', data);
```

Comment:
`(console.trace || console.error)(...)` doesn't work:

![image](https://cloud.githubusercontent.com/assets/98601/16752352/6a0b5c78-4795-11e6-9d6c-215ec9423243.png)

You have to use `call` or `apply`.